community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/input/csv/DataFactories.java

```diff
@@ -293,21 +297,26 @@ public boolean isDefined()
             String groupName = null;
 
             int typeIndex;
-            if ( rawHeaderField != null && (typeIndex = rawHeaderField.lastIndexOf( ':' )) != -1 )
-            {   // Specific type given
-                name = typeIndex > 0 ? rawHeaderField.substring( 0, typeIndex ) : null;
-                type = rawHeaderField.substring( typeIndex + 1 );
-                int groupNameStartIndex = type.indexOf( '(' );
-                if ( groupNameStartIndex != -1 )
-                {   // Specific group given also
-                    if ( !type.endsWith( ")" ) )
-                    {
-                        throw new IllegalArgumentException( "Group specification in '" + rawHeaderField +
-                                "' is invalid, format expected to be 'name:TYPE(group)' " +
-                                "where TYPE and (group) are optional" );
+
+            if ( rawHeaderField != null )
+            {
+                String rawHeaderUntilOptions = rawHeaderField.split( "\\{" )[0];
```

Comment:
Does that mean the header must not contain `{` anywhere before the new options? Can we always guarantee that?

Reply:
Please correct me if I am wrong but as far as I see it, ```{``` cannot be part of a variable name or a label and is not part of a data type. Thus it should be safe to use it that way.

Reply:
Seems correct to me.